inc/Engine/Preload/Admin/Settings.php

```diff
@@ -128,10 +128,11 @@ private function is_valid_as_tables() {
 		}
 
 		global $wpdb;
-
+		$exp = "'^" . $wpdb->prefix . "actionscheduler_+(logs|actions|groups|claims)$'";
```

Comment:
there is a conflict here with develop
`is_valid_as_tables` function are not in develop anymore